lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -134,8 +134,8 @@ class OffscreenImages extends ByteEfficiencyAudit {
       });
 
       const headings = [
-        {key: 'preview', itemType: 'thumbnail', text: ''},
-        {key: 'url', itemType: 'url', text: 'URL'},
+        {key: 'preview', itemType: 'thumbnail', text: '', styleAs: 'preview'},
+        {key: 'url', itemType: 'url', text: 'URL', styleAs: 'url'},
```

Comment:
what's the difference between styling as URL and type URL? Can we make them more orthogonal so it's clearer how/when to use one or the other or both?